### PR TITLE
Adding new code for bulk update post

### PR DESCRIPTION
**Pull request by choran**
Updated copy in new repo with changes from origin PR

### CODE REVIEW

File: bulk_update/bulk_update.rb [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+      user_data[:id] = user.id
+      # Create/update the user with a custom attribute
+      user_data[:custom_attributes] = cus_attr
+      #Put it all in an array so we can bulk create it when it reaches 100
+      all_users << user_data
+    end
+    scroll_num = result.records.length
+    scroll_param = result.scroll_param
+    # Check to see if the most recent request is empty (no more users to update)
+    if scroll_num > 0
```
Comment (kant01ne) [inline bulk_update/bulk_update.rb]: `return unless scroll_num > 0`
or `break unless scroll_num > 0`

File: bulk_update/bulk_update.rb [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+      bulk = intercom.users.submit_bulk_job(create_items: all_users)
+      puts("Bulk Request Number #{count}")
+      redis_store(bulk.id)
+      puts(bulk.id)
+      count+=1
+    end
+  end
+end while scroll_num > 0
```
Comment (kant01ne) [inline bulk_update/bulk_update.rb]: end while true

File: bulk_update/bulk_update.rb [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+MAX_RETRIES = 3
+SCROLL_RESET_TIME = 60
+scroll_num = 0
+# Set scroll param to nil initially and then populate with returned parameter
+scroll_param = nil
+count=1
+# Create new instance that you can use to store IDs
+$redis = Redis.new
+
+def check_attrib(attrib)
```
Comment (kant01ne) [inline bulk_update/bulk_update.rb]:
```
def has_avatar_cda(user)
  if user.avatar.image_url.nil?
   ...
   return user
end
```

**Comment by choran** [comment]
Made changes and discussed with Kevin so ok to merge

**Pull request merged**